system/Router/Router.php

```diff
@@ -462,15 +459,12 @@ public function autoRoute(string $uri)
     {
         $segments = explode('/', $uri);
 
+        // WARNING: Directories get shifted out of the segments array.
         $segments = $this->scanControllers($segments);
 
-        // If we don't have any segments left - try the default controller;
-        // WARNING: Directories get shifted out of the segments array.
-        if (empty($segments)) {
-            $this->setDefaultController();
-        }
+        // If we don't have any segments left - use the default controller;
```

Comment:
I think this comment should also be removed?

Reply:
It is true. I don't think to be removed.